### DISCUSSION TITLE
Ender 3 V2 Neo: Pulley rubber abrasion

### DISCUSSION THREAD

**Discussion by samuelT2** [mriscoc/Ender3V2S1#829] [Q&A] [answered]
Hi all

My Neo has a couple of dozens prints done and I just noticed some rubber coming of the pulleys (see pictures below). Is this...

a) ... normal and will be gone within a couple more prints? Though that's nasty because the rubber will fall onto the bed.
b) ... not normal, something malfunctions and needs replacement?
c) ... not normal, and I should loosen the screws a bit (see yellow arrow in the first picture)?

The E3V2 which didn't came pre-assembled didn't have that problem since I had to finetune the screws for myself.

Someone else has similar problems with the Neo?

![IMG_2034](https://user-images.githubusercontent.com/10983143/228461594-b2f4e088-ef53-4f30-8e9e-41f359d43707.jpg)
![IMG_2035](https://user-images.githubusercontent.com/10983143/228461600-c561798b-e6e4-4ece-9673-5715116d701b.jpg)

**Comment by TheWebMachine** [comment, marked as answer]
I had to tighten the rollers on my Ender 3 Plus and experienced some weardown of the rubber as the rollers reshaped to the actual angle of the rail.

Yes, it's annoying and will continue for a couple dozen hours of printing until the rollers settle in. These particles have nearly never fallen into a print. They tend to stay statically clung to the rollers.

I used a dry paper towel to collect the particles before every print by **slowly** moving the hotend back and forth with the paper towel against the roller. Static elec makes the bits collect on the paper. 

That said, you should adjust the single roller such that there is no wiggle in the roller to rail contact, but not too tight, to minimize weardown. 

Once the rollers settle in, I revisit the rollers...clean and adjust tightness (if needed) every 100 print hours or so. I also found the Y/bed rollers needed adjustment from time to time, too. 

The sad truth is: Rollers do dampen some vibration for better print quality and quieter operation, but come at the cost of wearing down some over time and needing semi-frequent adjustments to maintain precise position.

**Comment by samuelT2** [comment]
That's all I needed to know, thank you @TheWebMachine.
